hledger/package.yaml

```diff
@@ -104,6 +104,7 @@ dependencies:
 - safe >=0.2
 - shakespeare >=2.0.2.2
 - split >=0.1
+- statistics
```

Comment:
We've already discussed this in IRC; adds build time and more risk of hitting transient ecosystem breakages for users installing hledger, but it's a mainstream well-maintained lib that seems worth having in our toolbox.